docs/install.md

```diff
@@ -8,6 +8,15 @@ From Homebrew:
 brew update && brew install kops
 ```
 
+Development Releases and master can also be installed via Homebrew:
```

Comment:
Should we just add a link for developers to the compile and homebrew instructions?  I would like it to be another click for a developer to install, and not have people installing HEAD, unless they know they are now a developer.

Reply:
@chrislovecnm That's a good idea, I'll move this to a docs/development/install.md and link?

Reply:
Or I can add it in docs/development/building.md as an "easier" way to build and install?